website/addons/osfstorage/model.py

```diff
@@ -136,139 +102,131 @@ def create_waterbutler_log(self, auth, action, metadata):
         pass
 
 
-class BaseFileObject(StoredObject):
+class OsfStorageFileNode(StoredObject):
```

Comment:
Could you add a docstring that clarifies the disambiguates this model vs. `OsfStorageFileGuid` vs. `OsfStorageFileVersion`?